tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -338,11 +338,30 @@ var AppInstallManager = {
       this.systemBanner.show({
         id: 'preview-app-hint'
       });
+      this.previewOpened[manifestURL] = ++this.previewOpened[manifestURL] || 1;
     };
 
     var handlePreviewTerminated = () => {
       window.removeEventListener('previewterminated', handlePreviewTerminated);
-      this.uninstallPreviewApp();
+      var askAddPrompt = this.previewOpened[manifestURL] === this.openedTimes;
```

Comment:
Question: 
- Is it correct that we show the prompt only when the previewing counter exactly equals to `openedTimes`? -- or every time when the counter is greater than `openedTimes`? 
- Should we reset the counter?